### PR TITLE
Now using the passed props on each component

### PR DESCRIPTION
**Pull request by EthanHogan**
@tylerstanczak check this out when you can! thanks! 

Updated each component to now actually render the props that are passed down from App.

### CODE REVIEW

File: client/components/App.jsx [hunk 1]
```diff
@@ -35,18 +35,23 @@ class App extends React.Component {
...
         <Price
           price = {this.state.price}
         />
+        <br></br>
         <Logistics
           shipping = {this.state.shipping}
         />
+        <br></br>
         <Guarentee />
+        <br></br>
```
Comment (tylerstanczak) [inline client/components/App.jsx]: Love the breaks here I'm sure it adds wonderful spacing to your component. Very clever of you to make use of the br tag here, I respect that a lot. Another one, another one, ano-

File: client/components/Head.jsx [hunk 2]
```diff
@@ -1,7 +1,22 @@
...
+        {numRatings}
+    </div>
 );
 
+Head.propTypes = {
+  name: PropTypes.string.isRequired,
+  rating: PropTypes.number.isRequired,
+  numRatings: PropTypes.number.isRequired,
+};
+
```
Comment (tylerstanczak) [inline client/components/Head.jsx]: Way to assert your prop types, bro. I love that.

File: client/components/Subhead.jsx [hunk 3]
```diff
@@ -1,7 +1,21 @@
...
+        {condition}
+        <br></br>
+        {sold}
+        <br></br>
+        {availability}
+    </div>
 );
 
+Subhead.propTypes = {
+  condition: PropTypes.bool.isRequired,
```
Comment (tylerstanczak) [inline client/components/Subhead.jsx]: Is the condition really going to be a boolean and not a string? Just confirming here, I think a boolean for used/new would be fine.

Comment (EthanHogan) [reply client/components/Subhead.jsx]: Yeah the way our data was setup it is going be a boolean. True meaning free shipping, false meaning.. not free. lol

**Pull request merged**